packages/accounts/src/light-account/account.ts

```diff
@@ -32,45 +37,72 @@ import {
 } from "./utils.js";
 
 export type LightAccount<
+  TEntryPointVersion extends EntryPointVersion,
```

Comment:
LightAccount should default to 0.6. Actually, it shouldn't just default but require it for now because LightAccount doesn't support 0.7